pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -182,6 +193,15 @@ public void runSegmentMetrics() {
         }
         if ((externalView == null) || (externalView.getStateMap(partitionName) == null)) {
           // No replicas for this segment
+          TableType tableType = TableNameBuilder.getTableTypeFromTableName(tableName);
+          if ((tableType != null) && (tableType.equals(TableType.OFFLINE))) {
+            OfflineSegmentZKMetadata segmentZKMetadata =
+                ZKMetadataProvider.getOfflineSegmentZKMetadata(propertyStore, tableName, partitionName);
+            if (segmentZKMetadata != null && segmentZKMetadata.getPushTime() > System.currentTimeMillis() - waitForPushTime) {
```

Comment:
Isnt one hour a bit too much for push time for a segment? That can happen only if servers are down, in which case we want the alert, right? I would set it to 5 minutes, maybe.

Unless I am missing something